R/climdex.R

```diff
@@ -0,0 +1,126 @@
+#' climdex
```

Comment:
As with #19 can we have a more informative title - perhaps "Calculate climate indices"?